src/functions-and-arrays.js

```diff
@@ -29,9 +79,46 @@ const wordsUnique = [
   'bring'
 ];
 
```

Comment:
An alternative solution is `
const uniquifyArray = words => {
  let uniqueArr = [];
  for (let word of words) {
    if (!uniqueArr.includes(word)) uniqueArr.push(word);
  }
  return uniqueArr;
};`